### PR TITLE
fix: the command `arctl gen_test_dataset` recursively reads files

### PR DESCRIPTION
**Pull request by dayuy**
#### What type of PR is this?
bug

#### What this PR does / why we need it
the command `arctl gen_test_dataset` recursively reads files
#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

**Comment by 0xff-dev** [comment]
run: `make prepare-push`

**Comment by bjwswang** [comment]
@dayuy this should be a feature.

File: pkg/arctl/eval.go [hunk 1]
```diff
@@ -142,19 +143,20 @@ func EvalGenTestDataset(home *string, namespace *string, appName *string) *cobra
...
-			files, err := os.ReadDir(inputDir)
-			if err != nil {
-				log.Fatal(err)
-			}
-			for _, file := range files {
-				if file.IsDir() || filepath.Ext(file.Name()) != ".csv" || strings.HasPrefix(file.Name(), "ragas-") {
-					continue
+			err = filepath.WalkDir(inputDir, func(path string, d fs.DirEntry, err error) error {
+				if err != nil {
+					log.Fatal(err)
```
[the remark this answers is not in the document]
Comment (bjwswang) [reply pkg/arctl/eval.go:148]: let's just do return

File: pkg/arctl/eval.go [hunk 2]
```diff
@@ -168,14 +170,21 @@ func EvalGenTestDataset(home *string, namespace *string, appName *string) *cobra
...
 				if err != nil {
+					log.Fatal(err)
 					return err
 				}
+				return nil
+			})
+
+			if err != nil {
+				log.Fatal(err)
+				return err
```
Comment (bjwswang) [inline pkg/arctl/eval.go:187]: same here. Just return error.

File: pkg/arctl/eval.go [hunk 3]
```diff
@@ -168,14 +167,19 @@ func EvalGenTestDataset(home *string, namespace *string, appName *string) *cobra
...
-					filepath.Join(inputDir, file.Name()),
+					path,
 					evaluation.WithQuestionColumn(questionColumn),
 					evaluation.WithGroundTruthsColumn(groundTruthsColumn),
 					evaluation.WithOutput(output),
 				)
 				if err != nil {
 					return err
 				}
+				return nil
```
Comment (0xff-dev) [inline pkg/arctl/eval.go:178]: Just return, no need to judge err.

**Review by 0xff-dev — APPROVED** [review]

**Review by bjwswang — APPROVED** [review]

**Comment by bjwswang** [comment]
Thanks @dayuy !

**Pull request merged**